case/infrasim/virtual_node/T33249_idic_KCSLanPrint.py

```diff
@@ -0,0 +1,102 @@
+                               'response of ipmitool is:\n{}'.
+                        format(self.test_node.get_name(), str_rsp))
+
+        try:
+            node_lan_channel = self.data[fru['Product Name']]
+
+        except KeyError, e:
+            self.result(BLOCK,
+                        """
+                        KeyError: {}.
+                        Please supplement product name of node ({}) and the corresponding lan channel in {}.json.
+                        For more details, please read the document: https://infrasim.readthedocs.org/en/latest/
+                        """
+                        .format(e, e, self.__class__.__name__))
+        else:
+            # Mount disk vmdk to virtual node
+            # Boot all nodes QEMU host to disk
+            try:
+                self.stack.boot_to_disk(self.test_node,
+                                        disk_image=self.data['disk_image'])
+            except Exception, e:
+                self.result(BLOCK, 'Fail to boot node to disk: {}'.format(e))
+                return
+            try:
+                qemu_conn = self.stack.get_host_ssh(self.test_node,
+                                                    username=self.data[
+                                                        'host_username'],
+                                                    password=self.data[
+                                                        'host_password'],
+                                                    dhcp_server=self.data[
+                                                        'dhcp_server'],
+                                                    dhcp_user=self.data[
+                                                        'dhcp_username'],
+                                                    dhcp_pass=self.data[
+                                                        'dhcp_password'])
+            except Exception, e:
+                self.result(BLOCK, 'Fail to get host access: {}'.format(e))
+                return
+
+            qemu_conn.remote_shell(
+                'echo {} | sudo -S modprobe ipmi_devintf'.format(
+                    self.data['host_password']))
+            qemu_conn.remote_shell('echo {} | sudo -S modprobe ipmi_si'.format(
+                self.data['host_password']))
+
+            rsp = qemu_conn.remote_shell(
+                'echo {} | sudo -S ipmitool lan print {}'.format(
+                    self.data['host_password'], node_lan_channel))
+
+            self.log('INFO', 'rsp: \n{}'.format(rsp))
+
+            if rsp['exitcode'] == 0:
+                self.log('INFO',
+                         'Able to issue local IPMI command ipmitool lan')
+            else:
+                self.result(FAIL,
+                            'Failed to issue local IPMI command ipmitool lan, return: {}'.format(
+                                rsp))
+
+    def deconfig(self):
+        self.log('INFO', 'Deconfig')
```

Comment:
@helloarys you need to umount this disk image in deconfig. Or sub-sequence test may random another node while disk image is locked by this test_node.